.devcontainer/devcontainer.json

```diff
@@ -64,11 +64,13 @@
 				],
 			}
 		],
+		"python.terminal.activateEnvironment": true, // Activate Python Environment in Terminal created using the Extension.
 		"python.analysis.extraPaths": [
 			"/workspaces/magma/orc8r/gateway/python/",
 			"/workspaces/magma/lte/gateway/python/",
+			"/home/vscode/build/python/lib/python3.8/site-packages",
 		],
-		"python.defaultInterpreterPath": "/usr/bin/python",
+		"python.defaultInterpreterPath": "/home/vscode/build/python/bin/python3.8",
```

Comment:
If this needs to be in sync with the entry line 382 from file above, we should probably add a comment in both places to indicated this coupling (if there is no chance to enforce it).

Reply:
I looked into ways to enforce it, but doesn't seem to possible currently. Will add comments.

Some more detail: It seems like passing Docker environment variables to settings like analysis.extraPaths and the like is not possible. They can be referenced by `${containerEnv:SOME_VARIABLE}` but only inside the `remoteEnv` property (see [docs](https://code.visualstudio.com/docs/remote/devcontainerjson-reference#_variables-in-devcontainerjson) here) if I understand correctly. This property makes the variables available in terminals and for VSCode. But that doesn't mean they can be referenced as environment variables inside devcontainer.json.  (It is even not supported to use them in a settings.json, which could have been a possible workaround, see [this issue](https://github.com/microsoft/vscode/issues/2809#issuecomment-1031011012).)

Inside settings `python.defaultInterpreter` only predefined VSCode variables seem to be supported. (But the documentation I saw was not too explicit about this, see for example  [python.defaultInterpreter](https://code.visualstudio.com/docs/python/settings-reference#_general-settings) and [predefined variables](https://code.visualstudio.com/docs/editor/variables-reference).)

Reply:
Thanks for the detailed research and explanation! :pray: 

Doing such things across different files with different technologies is always a big hassle and often plainly not possible. Expecting something like it was the intention of suggesting simple comments in this case. Thanks for checking better methods anyway. :+1: